data/schema/schema_v1.json

```diff
@@ -646,7 +646,6 @@
                     "additionalProperties": false,
                     "required": [
                       "id",
-                      "title",
```

Comment:
You also need to add validation to say that if it is specified then it can't be empty.